src/pages/admin/orders/add/index.tsx

```diff
@@ -1,8 +1,10 @@
 import AdminLayout from "../../../../layout/admin";
 import "./styles.css";
 
-function AdminAddProductPage() {
-  return <AdminLayout>AdminAddProductPage</AdminLayout>;
+function AdminAddOrderPage() {
+  return <AdminLayout>
+
```

Comment:
never, ever work or add a file ona feature you still not begin with and only push the work you are currently working on.